2021-05-19 11:37:51,206 INFO    MainThread:14952 [wandb_setup.py:_flush():69] setting env: {}
2021-05-19 11:37:51,207 INFO    MainThread:14952 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-19 11:37:51,207 INFO    MainThread:14952 [wandb_init.py:_log_setup():336] Logging user logs to C:\Users\Lorenzo\PycharmProjects\flow-forecast\notebooks\wandb\run-20210519_113751-144e1whk\logs\debug.log
2021-05-19 11:37:51,208 INFO    MainThread:14952 [wandb_init.py:_log_setup():337] Logging internal logs to C:\Users\Lorenzo\PycharmProjects\flow-forecast\notebooks\wandb\run-20210519_113751-144e1whk\logs\debug-internal.log
2021-05-19 11:37:51,209 INFO    MainThread:14952 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000021B38B9E548>
2021-05-19 11:37:51,210 INFO    MainThread:14952 [wandb_init.py:init():369] calling init triggers
2021-05-19 11:37:51,210 INFO    MainThread:14952 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-19 11:37:51,210 INFO    MainThread:14952 [wandb_init.py:init():418] starting backend
2021-05-19 11:37:51,211 INFO    MainThread:14952 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=spawn, using: spawn
2021-05-19 11:37:51,220 INFO    MainThread:14952 [backend.py:ensure_launched():123] starting backend process...
2021-05-19 11:37:51,354 INFO    MainThread:14952 [backend.py:ensure_launched():128] started backend process with pid: 17968
2021-05-19 11:37:51,355 INFO    MainThread:14952 [wandb_init.py:init():423] backend started and connected
2021-05-19 11:37:51,358 INFO    MainThread:14952 [wandb_init.py:init():465] updated telemetry
2021-05-19 11:37:51,358 INFO    MainThread:14952 [wandb_init.py:init():484] communicating current version
2021-05-19 11:37:52,284 INFO    MainThread:14952 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.30 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-19 11:37:52,289 INFO    MainThread:14952 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-19 11:37:52,677 INFO    MainThread:14952 [wandb_init.py:init():522] starting run threads in backend
2021-05-19 11:37:52,899 INFO    MainThread:14952 [wandb_run.py:_console_start():1524] atexit reg
2021-05-19 11:37:52,900 INFO    MainThread:14952 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.WRAP
2021-05-19 11:37:52,900 INFO    MainThread:14952 [wandb_run.py:_redirect():1414] Wrapping output streams.
2021-05-19 11:37:52,900 INFO    MainThread:14952 [wandb_run.py:_redirect():1438] Redirects installed.
2021-05-19 11:37:52,900 INFO    MainThread:14952 [wandb_init.py:init():546] run started, returning control to user process
2021-05-19 11:37:52,900 INFO    MainThread:14952 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-19 11:37:52,900 INFO    MainThread:14952 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-19 11:37:56,402 INFO    MainThread:14952 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-19 11:37:58,745 INFO    MainThread:14952 [wandb_watch.py:watch():39] Watching
2021-05-19 11:46:18,336 INFO    MainThread:14952 [wandb_config.py:__setitem__():141] config set save_path = model_save\19_May_202111_46AM_model.pth - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000021B37922BC8>>
2021-05-19 11:46:18,337 INFO    MainThread:14952 [wandb_run.py:_config_callback():749] config_cb save_path model_save\19_May_202111_46AM_model.pth None
2021-05-19 11:46:35,731 INFO    MainThread:14952 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-19 11:46:35,731 INFO    MainThread:14952 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-19 11:52:48,087 INFO    MainThread:14952 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 0
2021-05-19 11:52:48,089 INFO    MainThread:14952 [wandb_run.py:_restore():1466] restore
2021-05-19 11:52:48,393 INFO    MainThread:14952 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
  media_count: 95
}
pusher_stats {
  uploaded_bytes: 6260879
  total_bytes: 6260879
}

2021-05-19 11:52:48,918 INFO    MainThread:14952 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 95
}
pusher_stats {
  uploaded_bytes: 6260879
  total_bytes: 6443948
}

2021-05-19 11:52:49,598 INFO    MainThread:14952 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 95
}
pusher_stats {
  uploaded_bytes: 6443948
  total_bytes: 6443948
}

2021-05-19 11:52:49,710 INFO    MainThread:14952 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 95
}
pusher_stats {
  uploaded_bytes: 6443948
  total_bytes: 6443948
}

2021-05-19 11:52:49,818 INFO    MainThread:14952 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 95
}
pusher_stats {
  uploaded_bytes: 6443948
  total_bytes: 6443948
}

2021-05-19 11:52:50,974 INFO    MainThread:14952 [wandb_run.py:_show_summary():1771] rendering summary
2021-05-19 11:52:50,975 INFO    MainThread:14952 [wandb_run.py:_show_history():1809] rendering history
2021-05-19 11:52:50,976 INFO    MainThread:14952 [wandb_run.py:_show_files():1838] logging synced files
